packages/client/README.md

```diff
@@ -23,18 +23,29 @@ The Lectern Client provides developers TypeScript code tools to interact with Le
 ## Developer Examples
 ### Data Fetching
 
+Data fetching from a lectern service can be perfromed through the `rest` module:
+
 ```ts
 import * as lectern from '@overture-stack/lectern-client';
 
 const lecternUrl = 'http://lectern.example.com';
-const dictionaryName = 'dictionary-name';
-const currentVersion = "2.3";
-const previousVersion = "2.1";
+const dictionaryName = 'my-example-schemas';
+const dictionaryVersion = "2.3";
+
+const allAvailableDictionariesResult = await lectern.rest.listDictionaries(lecternUrl);
+const filteredByNameDictionariesResult = await lectern.rest.listDictionaries(lecternUrl, {name: dictionaryName});
+
+const exampleDicionaryResult = await lectern.rest.fetchDiff(lecternUrl, {name: dictionaryName, version: dictionaryVersion});
```

Comment:
is `fetchDiff` removed? description of PR says 

"Future work is needed to also include a getDiff or getSchema/getField. All the POST/PUT endpoints that modify data will also require auth integration, and these have been left for future tasks."